cpp/open3d/geometry/RGBDImageFactory.cpp

```diff
@@ -18,7 +18,9 @@ std::shared_ptr<RGBDImage> RGBDImage::CreateFromColorAndDepth(
         bool convert_rgb_to_intensity /* = true*/) {
     std::shared_ptr<RGBDImage> rgbd_image = std::make_shared<RGBDImage>();
     if (color.height_ != depth.height_ || color.width_ != depth.width_) {
-        utility::LogError("Unsupported image format.");
+        utility::LogError(
+                "RGB image size ({} {}) and depth image size mismatch ({} {}).",
+                color.height_, color.width_, depth.height_, depth.width_);
```

Comment:
Possibly you meant to put depth resolution right after "size" and not at the end of sentence, i.e. it should read
`RGB image size ({} {}) and depth image size ({} {}) mismatch.`

Furthermore, there are 2 more similar resolution checks in methods below:-
 - `CreateFromSUNFormat`
 - `CreateFromNYUFormat`

It would be good to update the log message there as well.